rust/datafusion/src/utils.rs

```diff
@@ -120,6 +143,7 @@ pub fn array_value_to_string(column: array::ArrayRef, row: usize) -> Result<Stri
         DataType::Time64(unit) if *unit == TimeUnit::Nanosecond => {
             make_string!(array::Time64NanosecondArray, column, row)
         }
+        DataType::List(_) => make_string_from_list!(column, row),
```

Comment:
Does it matter what data type the list is? Can lists of any type be converted into strings?

Reply:
This recursively calls `array_value_to_string` on the array items, so an unsupported type should give the appropriate error on the recursive call